### PR TITLE
issue #2: Mime type handling

### PR DESCRIPTION
**Pull request by voila**
Adds Content-type headers to static files and HTML document.

### CODE REVIEW

**Comment by voila** [comment]
Sorry, not too sure how to squash these duplicate commits

**Comment by hannesm** [comment]
I usually do an interactive rebase (`git rebase -i origin/master`)...

File: canopy_dispatch.ml [hunk 1]
```diff
@@ -28,7 +28,8 @@ module Make (S: Cohttp_lwt.Server) (C: V1_LWT.CONSOLE) (Disk: V1_LWT.KV_RO)
     let respond_html ~headers ~status ~content ~title =
       store.subkeys [] >>= fun keys ->
       let body = Canopy_templates.main ~config ~content ~title ~keys in
-      S.respond_string ~headers ~status ~body ()
+      let headers = Cohttp.Header.init_with "Content-Type" "text/html; charset=UTF-8" in
```
Comment (hannesm) [inline canopy_dispatch.ml:31]: Here, we'd now (since there's already a `headers` flowing in) use `Cohttp.Header.add headers` instead... (same below)

**Comment by voila** [comment]
I have tried rebasing but I get this... 

```
$ git rebase -i origin/master
[detached HEAD 4d827df] issue #2: Mime type handling
 Date: Fri Apr 15 10:08:19 2016 +1200
 1 file changed, 2 insertions(+), 2 deletions(-)
error: could not apply a4ad6a3... issue #2: Mime type handling

Could not apply a4ad6a356ecaab6b3a4cdf06072c0b00a4f9b669... a4ad6a3
```

**Comment by hannesm** [comment]
hmmm... in that case, maybe a checkout of master, and manually applying the diff? :)

**Comment by voila** [comment]
I think it's because I have merged your changes in between the commits I am trying to squash... I am not sure. 

What do you mean by a checkout of master ? My fork has all these commits I can't rebase.... Do you mean I should fork again the original repo and apply my changes ?

Thanks for your help, my git fu is weak

**Pull request closed without merging**